app_dart/lib/src/model/ci_yaml/ci_yaml.dart

```diff
@@ -97,4 +118,74 @@ class CiYaml {
 
     return regexp.hasMatch(branch);
   }
+
+  /// validate [schedulerConfig], extracted from ci.yaml files provided by user
```

Comment:
In this case, it would be clearer to refer to the data structure instead of the instance

[schedulerConfig] -> [pb.SchedulerConfig]